src/components/Profile.js

```diff
@@ -71,7 +72,7 @@ class Profile extends Component {
                         <Form>
                             <Form.Input
                                 name="name"
-                                value={this.state.name ? this.state.name : userinfo.name}
+                                value={this.state.name}
```

Comment:
Even when the name exists in the backend, this shows as empty when tried to change, whereas the already existent name should be displayed.

Reply:
> 
> 
> Even when the name exists in the backend, this shows as empty when tried to change, whereas the already existent name should be displayed.

@bismitaguha, actually for the first time when we load the site the name disappears, but from next time it's appearing.